docs/manual_test_plan.md

```diff
@@ -18,3 +18,67 @@ requests are not turned on in production.
 |5|In the date range field, type `12/15/2026 - 1/13/2027`|The form shows the date range that you entered, and the 123.25 hours.|
 |6|In the date range field, use the date picker to enter a different date range|The form shows the new date range and the correct number of hours.|
 |7|Press submit request|You see "Absence request was successfully created."|
+
+### Travel request workflow
+
+Note: these steps must be completed on a non-production environment, to prevent corrupting production data.
+
+#### Creating a travel request
+
+|Step|Action|Expected result|
+|---|---|---|
+|1|Without an active session, go to [the application](https://approvals-staging.princeton.edu/)|The application has a button to Login with NetID|
+|2|Press "LOGIN with NetID" button and login with CAS credentials|You are taken to the My Requests page|
+|3|Press "New travel request" button|You see a form with even info and expenses|
+|4|In the Event Name field, type "Test Event 360"|Below the Event Name field you will see a message saying there appears to be a date in the field|
+|5|In the Location field, type "Princeton, NJ"|The form will show the entered value|
+|6|Select an Event Format|The form shows that you have selected the appropriate Event Format|
+|7|Select a Participation type|The form shows that you have selected the appropriate Participation type|
+|8|In the Purpose field, type "Training"|The form will show the appropriate Purpose|
+|9|In the Event Dates field, type "12/14/2026 - 12/18/2026"|The form will show the selected date range|
+|10|In the Travel Dates field, type "12/13/2026 - 12/19/2026"|The form will show the selected date range|
+|11|In the Notes to Approvers field, Type "This is a note"|The form will show the entered note|
+|12|In the Anticipated Expenses section, enter some expenses|The form will show a total for each line and an overall total|
+|13|Press the Submit Request button|A modal will appear indicating a possible date in the Event Name|
+|14|Press the Continue button|You see "Travel request was successfully created."|
```

Comment:
Can you add a test for the Anticipated Expences?

Reply:
Step 12 has a test for this, were you wanting a test with specific values?

Reply:
👍🏻  for step 12